src/runtime.cpp

```diff
@@ -645,6 +645,16 @@ namespace hpx
               &performance_counters::default_counter_discoverer,
               ""
             },
+            // arithmetics count counter
+            { "/arithmetics/count", performance_counters::counter_aggregating,
+              "returns the count value of all values of the specified "
+              "base counters; pass the required base counters as the parameters: "
+              "/arithmetics/count@<base_counter_name1>,<base_counter_name2>",
```

Comment:
If the counter returns the number of values in the accumulator, wouldn't it always return the number of base-counters specified as its argument?

Reply:
@hkaiser Thanks for the review 😊 . Actually , Yes it will return the number of base-counters . Sorry for being little unclear . As mean, median and variance are done on the base counter values . Similarly ```count``` is also there to return the count values .